### ISSUE TITLE
sdkjs: TypeError: e.indexOf is not a function

### ISSUE THREAD

**Issue by decentral1se** [ONLYOFFICE/DocumentServer#676]
**Do you want to request a *feature* or report a *bug*?**

Bug.

**What is the current behavior?**

When editing a cell, I see "An error occurred ..."

From the logs I see:

```
[2019-10-29T12:31:00.106] [ERROR] nodeJS - changesError: docId = 2096958959 Error: TypeError: e.indexOf is not a function Script: https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all-min.js Line: 660:213 userAgent: Mozilla/5.0 (X11; Linux x86_64; rv:69.0) Gecko/20100101 Firefox/69.0 platform: Linux x86_64 isLoadFullApi: true isDocumentLoadComplete: true StackTrace: db.prototype.BIc@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all-min.js:660:213
db.prototype.nP@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all-min.js:659:223
La.prototype.Awa@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:6841:272
d.prototype.vLg@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:7313:447
Mrg@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:7828:443
d.prototype.close@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:7392:122
d.prototype.Ieg@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:7437:370
d.prototype.sNe@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:7440:372
h.prototype.Y1/this.Zc.dga@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:7506:248
dga@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:11758:286
se/this.wo.onkeydown@https://mysite.org/5.4.1-39//sdkjs/cell/sdk-all.js:11742:220
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

Upload [Supplier Database.xlsx](https://github.com/ONLYOFFICE/DocumentServer/files/3783556/Supplier.Database.xlsx) and edit a cell. This is a copy of the file that I am trying to edit.

**What is the expected behavior?**

Editing the cell is successful.

**Did this work in previous versions of DocumentServer?**

No. I did just run an upgrade to latest (from Debian package repositories). I also downloaded the latest sdkjs release and compiled it and tried it but that led to a lot of other errors.

**DocumentServer version:** 

5.4.1

**Operating System:**

Debian Stretch

**Browser version:**

Firefox 69.X

**Comment by Rita-Bubnova** [comment]
Hello, @decentral1se.
Thank you. I can confirm - this is a bug, issue 43400 in our internal issue tracker.

**Comment by decentral1se** [comment]
Good stuff, looking forward to seeing a fix, thank you all for the good work :rocket:

**Comment by Rita-Bubnova** [comment]
This issue has been fixed (https://github.com/ONLYOFFICE/sdkjs/commit/8e2ff9663b45433d2c0ed08769474a5ed503949b) and will be released in future major release.

**Comment by decentral1se** [comment]
Hi @Rita-Bubnova, thanks a lot. When is the roadmap for the next major release?

**Comment by trofim24** [comment]
We fixed this in version 5.5.0
